man/systemd-sysusers.xml

```diff
@@ -94,6 +94,12 @@
         paths. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--direct</option></term>
+        <listitem><para>Treat each positional argument as a separate configuration
```

Comment:
what's a "positional" argument? I don't think anyone without a degree in advanced getoptology knows the name for that concept ;-)

Reply:
maybe call this option "--cmdline" or so, to indicate that this is about reading data from the cmdline?

Reply:
hmm, or maybe, instead, why do this as positional params at all? I think it would be more obvious to just do that as regular switch arguments, i.e.:

```
# systemd-sysusers --extra="u waldi" --extra="u quux"
````
I think that would be much nicer to read and avoids any confusion around getopt's requirement of adding "--" if you want structured further arguments...

And semantically it avoids some pitfalls regarding naming, as a switch like "--extra=" is more explanatory than "--cmdline" or "--direct", as it explains not how you define things, but simply that you define them here, and the "what" is like every other usual argument...